src/keylessBackup/encryption.ts

```diff
@@ -51,15 +51,15 @@ export function deriveKeyFromKeyShares(
 export function getSecp256K1KeyPair(
   keyshare1: Buffer,
   keyshare2: Buffer
-): { privateKey: Uint8Array; publicKey: Uint8Array } {
+): { privateKey: Hex; publicKey: Uint8Array } {
   const derivedKey = deriveKeyFromKeyShares(keyshare1, keyshare2, 48) // 40 is the minimum for hashToPrivateKey
   const privateKey = secp.utils.hashToPrivateKey(derivedKey)
   const publicKey = secp.getPublicKey(privateKey, true)
-  return { privateKey, publicKey }
+  return { privateKey: fromBytes(privateKey, 'hex'), publicKey }
```

Comment:
we use hex everywhere throughout the code, cleaner to just convert here once

Reply:
could we return both in hex to be consistent?

Reply:
sure, that makes sense